CMakeLists.txt

```diff
@@ -103,6 +108,21 @@ set(bullet_GIT_REPOSITORY https://github.com/RobotLocomotion/bullet-pod.git)
 set(bullet_GIT_TAG 54d0751b50f86481549391961b209f06ac5035f9)
 set(bullet_IS_PUBLIC TRUE)
 set(bullet_IS_CMAKE_POD TRUE)
+set(libccd_GIT_REPOSITORY https://github.com/danfis/libccd.git)
+set(libccd_GIT_TAG v2.0)
+set(libccd_IS_PUBLIC TRUE)
+set(libccd_IS_CMAKE_POD TRUE)
+set(flann_GIT_REPOSITORY https://github.com/mariusmuja/flann.git)
+set(flann_GIT_TAG 1.8.1-src)
+set(flann_IS_PUBLIC TRUE)
+set(flann_IS_CMAKE_POD TRUE)
+#set(fcl_GIT_REPOSITORY https://github.com/flexible-collision-library/fcl.git)
+#set(fcl_GIT_TAG 0.4.0)
+set(fcl_GIT_REPOSITORY https://github.com/law12019/fcl.git)
```

Comment:
fcl was not finding ccd in drake.  A pull request to fcl has been initiated.

On Fri, Mar 18, 2016 at 12:34 AM, Michael Sherman <notifications@github.com>
wrote:

> In CMakeLists.txt
> <https://github.com/RobotLocomotion/drake/pull/1886#discussion_r56612409>:
>
> > @@ -103,6 +108,21 @@ set(bullet_GIT_REPOSITORY https://github.com/RobotLocomotion/bullet-pod.git)
> >  set(bullet_GIT_TAG 54d0751b50f86481549391961b209f06ac5035f9)
> >  set(bullet_IS_PUBLIC TRUE)
> >  set(bullet_IS_CMAKE_POD TRUE)
> > +set(libccd_GIT_REPOSITORY https://github.com/danfis/libccd.git)
> > +set(libccd_GIT_TAG v2.0)
> > +set(libccd_IS_PUBLIC TRUE)
> > +set(libccd_IS_CMAKE_POD TRUE)
> > +set(flann_GIT_REPOSITORY https://github.com/mariusmuja/flann.git)
> > +set(flann_GIT_TAG 1.8.1-src)
> > +set(flann_IS_PUBLIC TRUE)
> > +set(flann_IS_CMAKE_POD TRUE)
> > +#set(fcl_GIT_REPOSITORY https://github.com/flexible-collision-library/fcl.git)
> > +#set(fcl_GIT_TAG 0.4.0)
> > +set(fcl_GIT_REPOSITORY https://github.com/law12019/fcl.git)
>
> Why did you have to make your own fork? Is there a change that could be
> pushed back to the upstream fork so it could be used instead?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/RobotLocomotion/drake/pull/1886/files/49f7ea386fc717bcef0638447af4f60164e83280#r56612409>
>